test-cases/features/repair-based-operations/longevity-200GB-48h-Decommission.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 3100
```

Comment:
Why is this test 200GB/48h while the below two tests are 50GB/12h and 50GB/12h?

Reply:
It's king of a little bit different scenario, we use it to validate cases of data loss in more precise way.